sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/xml/TestCase.java

```diff
@@ -23,234 +23,76 @@
 import com.thoughtworks.xstream.annotations.XStreamAsAttribute;
 import com.thoughtworks.xstream.annotations.XStreamOmitField;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.thoughtworks.xstream.annotations.XStreamOmitField'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)